build/cybersecurity-mlp/build.py

```diff
@@ -86,14 +88,29 @@ def platform_to_shell(platform):
             build_cfg.DataflowOutputType.ESTIMATE_REPORTS,
             build_cfg.DataflowOutputType.BITFILE,
             build_cfg.DataflowOutputType.DEPLOYMENT_PACKAGE,
+            build_cfg.DataflowOutputType.STITCHED_IP,
         ],
-        save_intermediate_models=True,
```

Comment:
`save_intermediate_models` is set to `True` by default, so it is ok to remove it. Technically, this is a change that is a clean up rather than a necessary change for this PR.